src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -9200,20 +9200,25 @@ void CEEInfo::getFunctionFixedEntryPoint(CORINFO_METHOD_HANDLE   ftn,
 
     MethodDesc * pMD = GetMethod(ftn);
 
-    pResult->accessType = IAT_VALUE;
-
-
-#ifndef CROSSGEN_COMPILE
-    // If LDFTN target has [NativeCallable] attribute , then create a UMEntryThunk.
+    // If this method has the NativeCallableAttribute, then validate it
+    // adheres to the limitations.
     if (pMD->HasNativeCallableAttribute())
     {
-        pResult->addr = (void*)COMDelegate::ConvertToCallback(pMD);
```

Comment:
It may be a good idea to keep `COMDelegate::ConvertToCall` path for Windows x86, in the first iteration at least. I suspect that making this work for `Windows x86` is going to have a bunch of unique problems.

Reply:
Okay.

Reply:
Oh right... Windows - boo.